front/app/components/admin/Table/Thead.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+// styling
+import styled from 'styled-components';
+
+interface Props {
+  children: React.ReactNode;
+  background?: string;
+}
+
+const StyledTHead = styled.thead<{ background?: string }>`
+  background: ${({ background }) => background ?? 'white'};
```

Comment:
Could you use `colors.white` - not a practical different atm but for consistency, it's better.